service/utils.hpp

```diff
@@ -0,0 +1,64 @@
+//
+// Created by milerius on 05/03/19.
+//
+
+#pragma once
+
+#include <string>
+#include <fstream>
+#include "service_strong_types.hpp"
+
+namespace raven
+{
+  inline constexpr const table_ascii_st table_ascii{
+      R"(!"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz{|}~')"};
+  inline constexpr const std::size_t table_ascii_len = 94;
+
+  inline std::string random_string(std::size_t random_string_len = 32) noexcept
+  {
+      std::random_device rd;
```

Comment:
Shouldn't you reuse the the random_device and engine?

Reply:
this is the case, look inside the lambda of std::generate

Reply:
I meant across calls to `random_string`